### PR TITLE
Update az.storage version for preview failover support

### PR DESCRIPTION
**Pull request by markcowl**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
netsign build:https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/325/
sign build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1560/

**Comment by blueww** [comment]
@markcowl 

I find both of the following build, build out the netcore binary:
•	netsign build:https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/325/
•	sign build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1560/

 I can’t find .net binary, so only do sanity check for following netcore module:
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/325/artifact/src/Package/ 
•	Az.Storage 0.5.1-pPreview
•	Az.Profile 0.5.0

There are 2 blocking  issues as below. Both of them not happen in the last released Az 0.5.0 build.

Would you please help to look?
If we can’t fix them before end of Monday, could we first build .net preview module, and release .net preview module first?

Thanks!


1.	The New-AzStorageContext cmdlet can’t be found. I test in goth PSH5 and PSH6, both not work. 

PSH6 Error:
```
New-AzStorageContext : The 'New-AzStorageContext' command was found in the module 'Az.Storage', but the module could not be loaded. For more information, run 'Import-Module Az.Storage'.
At line:1 char:1
+ New-AzStorageContext -StorageAccountName azcopyperf1 -StorageAccountK ...
+ ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (New-AzStorageContext:String) [], CommandNotFoundException
+ FullyQualifiedErrorId : CouldNotAutoloadMatchingModule
```

PSH5 Error:
```
PS C:\WINDOWS\system32> New-AzStorageContext -StorageAccountName azcopyperf1 -StorageAccountKey ***
New-AzStorageContext : The term 'New-AzStorageContext' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ New-AzStorageContext -StorageAccountName azcopyperf1 -StorageAccountK ...
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (New-AzStorageContext:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
  

2.	In PSH5, the Storage Resource module will get error for JSON. PSH6 works well.

PSH6 correct: 
```
PS C:\WINDOWS\system32> get-AzureRMStorageAccount -ResourceGroupName $RGName -Name $AccountName

StorageAccountName ResourceGroupName PrimaryLocation SkuName     Kind      AccessTier CreationTime         ProvisioningState EnableHttpsTrafficOnly
------------------ ----------------- --------------- -------     ----      ---------- ------------         ----------------- ----------------------
weifailover7       weitest           centraluseuap   StandardLRS StorageV2 Hot        10/9/2018 6:23:03 AM Succeeded         False
```

PSH5 Error:

```
PS C:\WINDOWS\system32> get-AzStorageAccount -ResourceGroupName $RGName -Name $AccountName
get-AzStorageAccount : Could not load file or assembly 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified.
At line:1 char:1
+ get-AzStorageAccount -ResourceGroupName $RGName -Name $AccountName
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzStorageAccount], FileNotFoundException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Management.Storage.GetAzureStorageAccountCommand 
```

File: src/ResourceManager/Storage/Commands.Management.Storage/Az.Storage.psd1 [hunk 1]
```diff
@@ -182,10 +182,13 @@ PrivateData = @{
...
         # ReleaseNotes of this module
-        ReleaseNotes = 'Update release with latest service features and serialization fixes'
+        ReleaseNotes = '* Support failover Storage account
+    - Invoke-AzureRmStorageAccountFailover
+* Support Get last sync time of Storage account
+    - Get-AzureRmStorageAccountLastSyncTime'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```
Comment (cormacpayne) [inline src/ResourceManager/Storage/Commands.Management.Storage/Az.Storage.psd1]: @markcowl I thought for `0.x.x` versions of modules we weren't adding the `preview` suffix for preview modules -- I thought we were only adding the suffix for stable modules.

Comment (markcowl) [reply src/ResourceManager/Storage/Commands.Management.Storage/Az.Storage.psd1]: this is a preview version of a stable module

**Pull request merged**